pkg/gui/controllers/helpers/suggestions_helper.go

```diff
@@ -82,7 +83,7 @@ func (self *SuggestionsHelper) GetBranchNameSuggestionsFunc() func(string) []*ty
 		if input == "" {
 			matchingBranchNames = branchNames
 		} else {
-			matchingBranchNames = utils.FuzzySearch(input, branchNames)
+			matchingBranchNames = utils.FilterStrings(input, branchNames, self.c.UserConfig.Gui.FilterMode == "fuzzy")
```

Comment:
The line `self.c.UserConfig.Gui.FilterMode == "fuzzy"` is duplicated in a lot of places: what are your thoughts on centralising that somewhere in a function?

Reply:
I just followed the pattern that we use for all other config options that are strings. It didn't feel comfortable with that either, but wasn't sure where to put the function. Here's one option that adds a method to GuiConfig: 7718fb49c9b4. We could follow this pattern in other cases too if we wanted to.